### PR TITLE
Topic Psychological Quiz

### PR DESCRIPTION
**Pull request by TacticalTriangle**

### CODE REVIEW

**Comment by ThePotatoGuy** [comment]
Since I just did a merge of `content` x `master`, a base change is appropriate this time.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4790,7 +4790,78 @@ label monika_urgent:
...
+                "A truck full or more.":
+                    m "..."
+                    m "hehehe~"
+                    m "I can't believe you would take so many flowers I mean what is the owner supposed to do?"
+                    menu:
+                        "So what does my answer mean?":
+                            m "Huh? What does your answer mean?"
+                            m "The amount of flowers you took reveals the size of your love.~"
+                            m "I love you a truckload [player].~"
+        "2"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: This `menu` block choice is missing a colon.

**Comment by Winblows2** [comment]
Sorry for being overly critical, but read this sentence aloud;
m "I can't believe you would take so many flowers I mean what is the owner supposed to do?"

It seems like it could be spliced into two sentences or something else. Just a though, though.

**Comment by Winblows2** [comment]
I feel like a comma would go after "I mean":

"I mean, what is the owner supposed to do?"

**Comment by Winblows2** [comment]
Also, bad formatting. Closing for a rewrite.

**Pull request closed without merging**